tests/postgres_tests/migrations/0001_setup_extensions.py

```diff
@@ -22,9 +23,10 @@ class Migration(migrations.Migration):
         BtreeGinExtension(),
         # Ensure CreateExtension quotes extension names by creating one with a
         # dash in its name.
+        CITextExtension(),
```

Comment:
You're right I alphabetical ordered the operation but I forgot to move the comment too. I'm fixing it.